EventDisplay/fcl/EventDisplay.fcl

```diff
@@ -16,6 +16,7 @@ source :
 services :
 {
     GeometryService        : { inputFile : "Mu2eG4/geom/geom_common.txt" }
+    ConditionsService      : { conditionsfile : "Mu2eG4/test/conditions_01.txt" }
     GlobalConstantsService : { inputFile : "Mu2eG4/test/globalConstants_01.txt" }
```

Comment:
I am a little surprised this works.  iI thought that we needed the tool_type parameter in the GeometryService, ProditionsService and a few others.

If they are not needed now, they will be soon.

Consider using @local::Services.Core which is defined in fcl/standardServices.  This will minimize future maintenance.

Reply:
I will change it.

Reply:
I pushed the change.